### PR TITLE
New Catalonia source

### PR DESCRIPTION
**Pull request by stepps00**
Per https://github.com/whosonfirst/whosonfirst-sources/issues/149, this PR:

- Adds a new json file for Catalonian ICGC
- Updates spec docs

### CODE REVIEW

File: sources/README.md [hunk 1]
```diff
@@ -720,6 +720,20 @@ _Neighbourhood (district) data as provided by the Barcelona City Council (CartoB
...
+
+* added: `2019-05-21`
+* id: `1393679059`
+* name: `esp-cicgc`
+* prefix: `esp-cicgc`
+* license_type: _CC BY 4.0_
+* license_text: _You are free to: Share — copy and redistribute the material in any medium or format [...] Adapt — remix, transform, and build upon the material for any purpose, even commercially._
+* url: _https://github.com/OpenICGC/limits_administratius/tree/master/data_
+* license: _http://www.icgc.cat/en/The-ICGC/Public-Information/Transparency/Re-use-of-the-information_
+* usage: `property`, `geometry`
```
Comment (nvkelso) [inline sources/README.md]: Do they have any unique IDs we can bring over, too?

File: sources/README.md [hunk 2]
```diff
@@ -720,6 +720,20 @@ _Neighbourhood (district) data as provided by the Barcelona City Council (CartoB
 * license: _http://w133.bcn.cat/geoportal/descargas/en_gb_cond_us_carto.pdf_
 * usage: `property`, `geometry`
 
+## Institut Cartogràfic i Geològic de Catalunya
+
+_Official Site of the Institut Cartogràfic i Geològic de Catalunya (ICGC)_ 
+
+* added: `2019-05-21`
+* id: `1393679059`
+* name: `esp-cicgc`
```
Comment (nvkelso) [inline sources/README.md]: what's the leading `c` in `cicgc` for?

File: sources/README.md [hunk 3]
```diff
@@ -720,6 +720,20 @@ _Neighbourhood (district) data as provided by the Barcelona City Council (CartoB
 * license: _http://w133.bcn.cat/geoportal/descargas/en_gb_cond_us_carto.pdf_
 * usage: `property`, `geometry`
 
+## Institut Cartogràfic i Geològic de Catalunya
+
+_Official Site of the Institut Cartogràfic i Geològic de Catalunya (ICGC)_
```
Comment (nvkelso) [inline sources/README.md]: We don't usually say: `Official Site of the `

Comment (nvkelso) [reply sources/README.md]: (Should have commented on the JSON, but same difference...)

File: sources/README.md (hunk 1 above)
Comment (stepps00) [reply sources/README.md]: We can bring over the `la_lc` as a concordance (which is actually stored in a `qs:` property too).

**Comment by stepps00** [comment]
Most recent commits update the source json and re-run the makefile to update spec docs.. merging (and taking care of property issue in data repo).

**Pull request merged**